Library/Formula/libstfl.rb

```diff
@@ -18,6 +18,10 @@ class Libstfl < Formula
   patch :DATA
 
   def install
+    if Formula["ruby"].installed?
```

Comment:
@DomT4 And to answer your question, yes, as I mentioned here:
https://github.com/Homebrew/homebrew/issues/48930#issuecomment-182744898
Also, I'm not sure if this is a bug, but while it likes
```
args = ["RUBY=/usr/bin/ruby CC=#{ENV.cc} -pthread", "prefix=#{prefix}"]
```
it doesn't like
```
args = ["RUBY=/usr/bin/ruby", CC=#{ENV.cc} -pthread", "prefix=#{prefix}"]
```

However, I'm not sure this fix is desirable, since you end up with
`/usr/local/Cellar/libstfl/0.22/lib/ruby/2.0.0/universal-darwin15/stfl.bundle`
instead of
`/usr/local/Cellar/libstfl/0.22/lib/ruby/2.3.0/x86_64-darwin15/stfl.bundle`
even when you have brewed ruby.